### DISCUSSION TITLE
Set up end2end gymchrono-importable a-stack

### DISCUSSION THREAD

**Discussion by StefanCaldararu** [uwsbel/miniav#130] [Ideas]
Set up an ART a-stack so that we can grab gymchrono trained models and have them drive ART.

**Comment by StefanCaldararu** [comment]
@AaronYoung5 this should be fairly easy to do for a "base" version, but would also be nice to have this set up such that people can select what sensors they are using for training in gymchrono, and then the metapackage and launch file is automatically generated based on that (bcs the only nodes for this are sensor nodes and a control node). Thoughts on how to do that?

**Comment by AaronYoung5** [comment]
idk about "automatically generated", seems like a compilation type thing where we have a chrono sim -> ros stack which is probably out of scope. Probably a good use case for the ChronoAI thing. Am I understanding what you're saying correctly?

**Reply by StefanCaldararu** [reply]
Yeah that makes sense, for now I am just going to write something similar to what we had for `keras` but for `stablebaselines3` for the control node. I have a trained model for path following I wanted to test. Then the next application for this would be @Huzaifg 's target point model, which would likely only require a bit of modification of the path planning.

**Comment by AaronYoung5** [comment]
Gonna move to discussion.
